extras/utils.rb

```diff
@@ -19,7 +19,9 @@ def self.normalize(url)
     url.slice! %r{/Default\.aspx$}
 
     url.slice! %r{https?://} # consider http and https the same
-    url.slice! %r{^(www\d*\.)} # remove www\d* from domain
+    if url.split(".").size > 2 # only  removes www\d* if the url is not like www10.org, issue #1339
```

Comment:
You don't need to include the issue number in a comment, and your "if the url is not like www10.org" can go on the existing comment.

Please add two tests to `spec/extras/normalize_url_spec.rb`, one for `www10.org` and one for `www10.example.org`.